pypesto/optimize/optimizer.py

```diff
@@ -558,3 +563,50 @@ def minimize(
 
     def is_least_squares(self):
         return False
+
+
+class CmaesOptimizer(Optimizer):
+    """
+    Global optimization using cma-es.
+    """
+
+    def __init__(self, options: Dict = None):
+        super().__init__()
+
+        if options is None:
+            options = {'maxiter': 10000}
+        self.options = options
+        self.parameter_sigma0 = 0.25
+
+    @fix_decorator
+    @time_decorator
+    @history_decorator
+    def minimize(
+            self,
+            problem: Problem,
+            x0: np.ndarray,
+            id: str,
+            history_options: HistoryOptions = None,
+    ) -> OptimizerResult:
+        lb = problem.lb
+        ub = problem.ub
+        sigma0 = self.parameter_sigma0 * np.median(ub)
```

Comment:
The recommendation from the CMAES doc was 1/4 of the search domain width right?
Shouldn't it be `np.median(ub-lb)` then?
```suggestion
        sigma0 = self.parameter_sigma0 * np.median(ub-lb)
```

Reply:
I fixed it